### PR TITLE
VP-5929: Add API to return store URL

### PR DESCRIPTION
**Pull request by asvishnyakov**

### CODE REVIEW

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Web.Controllers.Api
+{
+    [Route("")]
+    public class DemoSolutionFeaturesModuleController : Controller
+    {
+        // GET: api/stores/url/{storeName}
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs]: Are we really need this comment here?

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Web.Controllers.Api
+{
+    [Route("")]
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs]: This attribute is required?

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs (hunk 1 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs]: Removed

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs (hunk 2 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoSolutionFeaturesModuleController.cs]: Removed

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-5995 has been created

**Review by pushnitsa — DISMISSED** [review]

**Review by pushnitsa — DISMISSED** [review]

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**